src/options.js

```diff
@@ -4,6 +4,7 @@ const validate = require("jest-validate").validate;
 const deprecatedConfig = require("./deprecated");
 
 const defaults = {
+  cursorOffset: -1,
```

Comment:
I could remove it from `defaults`, but then there's a validation warning:

```
  console.warn node_modules/jest-validate/build/utils.js:43
    ● Validation Warning:

      Unknown option "cursorOffset" with value 2 was found.
      This is probably a typing mistake. Fixing it will remove this message.
```

Not sure if there's a way to make it optional, so I'll leave it as-is for now. We can always revisit it later.